collections.md

```diff
@@ -90,6 +90,11 @@ Exposed links:
 * license: link to the license template used by the collection
 * defaultAccessConditions: link to the resource policies applied by default to new submissions in the collection
 * parentCommunity: the community containing this collection
+* adminGroup: the Collection Administrator group
+* submittersGroup: the Collection Submitters group
+* itemReadGroup: the Collection Default item READ rights group
+* bitstreamReadGroup: the Collection Default bitstream READ rights group
+* workflowGroups/<:workflow-role>: the Collection Workflow groups
```

Comment:
should we splt the workflowGroups in two subpaths one for "collection" groups and one for general groups? in the configurable workflow indeed it is possible to define roles that are repository wide and not specific of a collection. Split them over separate subpath will make easier the management of permission as collection admins are not expected to be always allowed to edit "repository wide" roles.
In addition or in alternative, the defined roles should be exposed in the workflow definition and maybe here we can flag the "repository", "collection" and "item" roles

Reply:
For DSpace 6, the edit collection page only displays the workflow groups with the collection scope.
This is indeed because only general admins can manage the repository-wide workflow groups

It's indeed possible to extend this to expose `collections/<:uuid>/collectionWorkflowGroups/<:workflow-role>` and `collections/<:uuid>/repositoryWorkflowGroups/<:workflow-role>` where only general admins can access the latter.
But the `collections/<:uuid>/repositoryWorkflowGroups/<:workflow-role>` would work quite differently since it can't be created or deleted. It would only be present to display the group.
For that reason, I would personally prefer to use [/api/config/workflowdefinitions/search/findByCollection?uuid=<:collection-uuid>](https://github.com/DSpace/Rest7Contract/blob/master/workflowdefinitions.md#findbycollection) and create an addition to expose the roles in the workflow definition or workflow step.

Reply:
The current approach (in this REST Contract) seems fine to me, along with what @benbosman notes.  If we find a reason to enhance/change this Contract in the future we can do so then.